.github/workflows/release-asset.yml

```
@@ -0,0 +1,57 @@
+name: Elixir CI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Build and publish release asset
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Elixir
+        uses: actions/setup-elixir@v1
+        with:
+          elixir-version: '1.10.4'
+          otp-version: '23.1'
```

This should be built with the oldest supported version of elixir:
```suggestion
          elixir-version: '1.8.2-otp-21'
          otp-version: '21.3.8.17'
```
Otherwise users will be unable to use ElixirLS if they're still using an older version. But ideally we could add multiple compiled versions and allow users to choose between them at runtime.

Also is there a way that this could use the versions specified in `.release-tool-versions` or vice-versa? Although maybe we'll just have to manually keep them in sync, in which case we should add a comment to that effect in both locations. `.release-tool-versions` will still be used when testing a release locally.